server/src/yml-objects/AbstractYmlObject.ts

```diff
@@ -82,26 +86,58 @@ export abstract class AbstractYmlObject implements CompletionItem {
         scopeStartOffset?: number,
         scopeEndOffset?: number,
     ): void {
-        sourceElementName = sourceElementName ? sourceElementName : 'static';
+        this.sourceElementName = sourceElementName ? sourceElementName : 'STATIC';
         this.data = `id_${sourceElementName}_${this.label}`;
-        this.setDocumentation(getDocumentation(fields, connection));
-        this.detail = getType(fields, connection, baseType);
+        const doc = getDocumentation(fields, connection);
+        this.setDetail(getType(fields, connection, baseType));
+        this.setUserInformations(this.detail, doc);
         if (scopeEndOffset && scopeStartOffset) {
             this.scopeEndOffset = scopeEndOffset;
             this.scopeStartOffset = scopeStartOffset;
         }
     }
 
+    public setDetail(type: string): void {
+        this.detail =
+            this.sourceElementName === 'STATIC'
+                ? `(${this.kindName}) [${this.sourceElementName}] ${this.label} ⇒ ${type}`
+                : `(${this.kindName}) [${this.sourceElementName}].${this.label} ⇒ ${type}`;
```

Comment:
I _literally_ took me more than ten seconds to spot the difference, man.

Reply:
Sorry :sob: